src/components/InitialComponents/LegislatorHomePage/index.js

```diff
@@ -0,0 +1,197 @@
+
+import Menu from "../../Menu";
+import HeaderBar from "../../HeaderBar/index";
+import LegislatorCard from "../../LegislatorCard/index";
+import ModalComponent from "../../Modal/index";
+import Modal from "../../ModalRequirements";
+
+import { mockRequirement } from "../../../service/api";
+
+
+export default function LegislatorHomePage() {
+
+    const modalRef = useRef();
+     
+    const [requirement, setRequirement] = useState(mockRequirement);
+
+    const [support, setSupport] = useState(false);
+    const [showComments, setShowComments] = useState(false);
+    const [comment, setComment] = useState("");
+
+    const handleSupport = () => {
+        setSupport(!support);
+        if (!support) setRequirement({ ...requirement, likes: requirement.likes + 1 });
+        // POST -> /support
+        else setRequirement({ ...requirement, likes: requirement.likes - 1 }); // POST -> /unsupport
+      }; 
+    
+    const handleShowComments = () => {
+        setShowComments(!showComments);
+    }
+
+    const handleComment = () => {
+        const data = {
+          profile: requirement.profile.photo,
+          name: requirement.profile.name,
+          message: comment,
+        };
+    
+        if (comment) {
+          setRequirement({
+            ...requirement,
+            comments: [...requirement.comments, data],
+          });
+          setComment("");
+        }
+      };
+
+    return (
+        <div>
+            <Menu />
+
+            <HeaderBar />
+                      
+            <LegislatorCard requeriments={299} analysis={274} denied={10}
+            done = {15}>  </LegislatorCard>
+
+            <section className="requirement_container"> 
+                <section  className="requirement_content"  onClick={() => modalRef.current.openModal()}>
+                    
+                    <div className="header_title">
+                        <p>Reivindicação em destaque</p>
+                        <Box className="status">
+                            <span>{requirement.status}</span>
+                        </Box>
+                    </div>
+                        
+                    <div className="header_profile">
+                                                             
+                            <Box className="profile">
+
+                                { requirement.profile.photo ? (
+                                <img
+                                    src={requirement.profile.photo}
+                                    width={30}
+                                    height={30}
+                                    alt="Image"
+                                />
+                                    ) : (
+                                    <AccountCircleRoundedIcon color="action" />
+                                )}
+
+                                <Box className="info">
+                                    <p>Nome: <strong>{requirement.profile.name}</strong></p>
```

Comment:
```suggestion
                                    <p>{requirement.profile.name}</p>
```
Na prototipagem não tem a predefinição "Nome:" e "Cidade:"

<img src="https://user-images.githubusercontent.com/56003521/159011202-628173ea-4354-4a07-9922-db0fa7ede47e.png" />